### DISCUSSION TITLE
Suggestion to watch World Cup 2022?

### DISCUSSION THREAD

**Discussion by davidanthonyn** [iptv-org/community#296] [Q&A] [answered]
I want to watch world cup from IPTV, but the channel that I know(in my country) has been delisted from IPTV. Any channels?

**Comment by pukozs75** [comment, marked as answer]
TVI Portugal
SIC Portugal
HRT 2 Croatia
Those are channels that i know.
And they are broadcasting World Cup.

**Reply by mabdallah2001** [reply]
Where do i find these? i tried opening them but didnt show anything

**Comment by BellezaEmporium** [comment]
> I want to watch world cup from IPTV, but the channel that I know(in my country) has been delisted from IPTV. Any channels?

If you have a VPN : 

TF1 (France) (account needed)
RTBF La Une (Belgium, has all matches)
TVI and SIC (Portugal)

and for paid channels beIN Sports.

**Comment by PD-TheKrazy0ne** [comment]
could add these to your playlist... they are showing it

#EXTINF:-1 tvg-logo="https://upload.wikimedia.org/wikipedia/pt/d/dc/Logotipo_TNT_Sports.png" group-title="DEPORTE", EVENTOS TNT Sports Brasil https://glxlmn026c.singularcdn.net.br/playout_01/playlist-720p.m3u8

#EXTINF:-1 tvg-logo="https://upload.wikimedia.org/wikipedia/pt/d/dc/Logotipo_TNT_Sports.png" group-title="DEPORTE", EVENTOS TNT Sports 2 Brasil https://glxlmn026c.singularcdn.net.br/playout_02/playlist-720p.m3u8

#EXTINF:-1 tvg-logo="https://upload.wikimedia.org/wikipedia/pt/d/dc/Logotipo_TNT_Sports.png" group-title="DEPORTE", EVENTOS TNT Sports 3 Brasil https://glxlmn026c.singularcdn.net.br/playout_03/playlist-720p.m3u8

#EXTINF:-1 tvg-logo="https://upload.wikimedia.org/wikipedia/pt/d/dc/Logotipo_TNT_Sports.png" group-title="DEPORTE", EVENTOS TNT Sports 4 Brasil https://glxlmn026c.singularcdn.net.br/playout_04/playlist-720p.m3u8

#EXTINF:-1 tvg-logo="https://upload.wikimedia.org/wikipedia/pt/d/dc/Logotipo_TNT_Sports.png" group-title="DEPORTE", EVENTOS TNT Sports 5 Brasil https://glxlmn026c.singularcdn.net.br/playout_05/playlist-720p.m3u8

**Reply by trecart** [reply]
Do u have some 1080p resources? thank u.

**Reply by BellezaEmporium** [reply]
basically remove "-720p" next to the name to have the adaptive one.
and no, they don't stream in 1080p.

**Reply by Nicolas1307** [reply]
Tyc sport argentina? O tn sport argentina porfa

**Comment by R4IN2007** [comment]
Use TyC Sports, Its Aired The World Cup
Here The m3u8
#EXTINF:-4, tvg-id=""  tvg-logo="logo" tvg-name="TyC Sports" group-title="Sports",TyC Sports
https://d3awnlgqz0szay.cloudfront.net/out/v1/b841c366cbe540e6a106c3ba83e5c8d6/index.m3u8

**Reply by sdefilippe** [reply]
Link doesn't work please help

**Reply by R4IN2007** [reply]
m3u8 stopped working :(

**Reply by R4IN2007** [reply]
m3u8 stopped working :(
Try This m3u8s
#EXTINF:-4, UBC | SBS UHD
http://59.23.231.102:1935/live/UBCstream/playlist.m3u8

#EXTINF:-4, TJB | SBS UHD
http://1.245.74.5:1935/live/tv/.m3u8

#EXTINF:-4, JIBS | SBS
http://123.140.197.22/stream/1/play.m3u8

**Reply by R4IN2007** [reply]
> Link doesn't work please help

BNT 1 [VLC ONLY]
http://85.14.32.15:8801/udp/239.100.102.32:8888

![20221205_130255](https://user-images.githubusercontent.com/83226646/205721459-7ed6f4ac-577e-4f6c-a525-02b8e9503c35.jpg)

**Reply by R4IN2007** [reply]
ITV 1
https://cdnuk001.broadcastcdn.net/KUK-ITV-107/tracks-v1a1/mono.m3u8

**Reply by BellezaEmporium** [reply]
Caracol HD http://201.244.106.58:8990/play/a00i/index.m3u8

**Reply by Nicolas1307** [reply]
Tyc sport m3u8 ???

**Reply by uwansm** [reply]
tyc sports (vlc only)
http://rctv.tech:80/EbhTbZp8Zudg/kQyPeRtp2jRM/1061397
http://megatitan.net:8080/Victorsepulveda53/VfmdXeFurCKB/451260

**Comment by R4IN2007** [comment]
Bein Sports 1 MAX
http://141.11.127.188:8080/memfs/67acb14d-9210-461b-a20f-13d08eb4857f_output_0.m3u8

**Reply by abdulforreal** [reply]
How about Bein MAX 2 for the second match
You have the link?

**Reply by R4IN2007** [reply]
> How about Bein MAX 2 for the second match You have the link?

Here A New Bein Sports MAX 1 And Bein Sports MAX 2

Bein Sports MAX 1
https://streamermytvc.com/Bein-Sp-Max-1-LB+ph===---/cache/Bein-Sp-Max-1-LB/master.m3u8

Bein Sports MAX 2
https://streamermytvc.com/Bein-Sp-Max-2-LB+ph===---/cache/Bein-Sp-Max-2-LB/master.m3u8

**Reply by abdulforreal** [reply]
Thank you
شكرا لك

**Reply by R4IN2007** [reply]
> Thank you شكرا لك

👍

**Reply by KABRIM-CRAZY** [reply]
> > How about Bein MAX 2 for the second match You have the link?
> 
> Here A New Bein Sports MAX 1 And Bein Sports MAX 2
> 
> Bein Sports MAX 1 https://streamermytvc.com/Bein-Sp-Max-1-LB+ph===---/cache/Bein-Sp-Max-1-LB/master.m3u8
> 
> Bein Sports MAX 2 https://streamermytvc.com/Bein-Sp-Max-2-LB+ph===---/cache/Bein-Sp-Max-2-LB/master.m3u8

It’s not working

**Reply by BellezaEmporium** [reply]
Works nicely on my side, Max 1 is showing Brazil - Croatia preparation.

**Reply by KABRIM-CRAZY** [reply]
> Works nicely on my side, Max 1 is showing Brazil - Croatia preparation.

IN VLC ??

**Reply by BellezaEmporium** [reply]
In HLS + DASH Player.

**Reply by KABRIM-CRAZY** [reply]
> In HLS + DASH Player.

OK THANK YOU BRO
